app/views/static_pages/formula.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="menu">
+  <a class="tab index tooltip active" href="/index" name="Projects">
+    <img src="<%= asset_path('leaf-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/attributes" name="Attributes">
+    <img src="<%= asset_path('edit-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/information" name="Information">
+    <img src="<%= asset_path('info-white.svg') %>">
+  </a>
+  <a class="tab logout tooltip" href="/.." name="Log Out">
```

Comment:
We usually prefer to use named routes in Rails for routing (eg. `root_path` instead of `/`). This is why we often use `link_to` to create links. For example, I think the first anchor tag could be written as:

```ruby
<%= link_to 'Projects', '/index', class: 'tab index tooltip active' do %>
  <%= image_tag 'leaf-white.svg' %>
<% end %>